### DISCUSSION TITLE
Is it possible to create an alarm application using ZeppOS v2.1?

### DISCUSSION THREAD

**Discussion by AlexandreFossati** [zepp-health/zeppos-forum#313] [Q&A] [answered]
I was reading the docs and I found this [alarm page](https://docs.zepp.com/docs/reference/device-app-api/newAPI/alarm/set/) that says the alarm feature is only available in `v3.0`.
Does it mean that I cannot develop a custom alarm in Amazfit Bip 5, which uses ZeppOS v2.1?
I was wondering if v2.1 has any other API which I can use to set alarms.

My wish is to have alarms with messages on my watch to set reminders. But I don't wanna rely on applications that depends on the smartphone through bluetooth connectivity, I want that app to be standalone and runs in the watch only.

**Comment by silver-zepp** [comment, marked as answer]
Yes, it's possible and here's how you can do it on `zos v2`: [setLaunchAppTimeout](https://docs.zepp.com/docs/v2/reference/device-app-api/newAPI/router/setLaunchAppTimeout)

**Reply by AlexandreFossati** [reply]
Thank you @silver-zepp, I will try this function.

**Reply by AlexandreFossati** [reply]
That works, thanks again!

**Reply by AlexandreFossati** [reply]
Hey @silver-zepp, is there a way to set the launch to be at a specific time? Let's say I want it to ring at 8AM, how can I do it?
Is there a way to create a `new Timer()` with parameters like the `new Date()` in js?

**Reply by silver-zepp** [reply]
Hey, yeah, we support [Date JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) out of the box, so you can get 8 AM for the next day from that and use it as a `utc` param.
